tools/packaging/static-build/cloud-hypervisor/build-static-clh.sh

```diff
@@ -48,7 +48,7 @@ build_clh_from_source() {
     info "Build ${cloud_hypervisor_repo} version: ${cloud_hypervisor_version}"
     repo_dir=$(basename "${cloud_hypervisor_repo}")
     repo_dir="${repo_dir//.git}"
-    [ -d "${repo_dir}" ] || git clone "${cloud_hypervisor_repo}"
+    [ -d "${repo_dir}" ] || git clone --depth 1 "${cloud_hypervisor_repo}"
```

Comment:
this may not work since we need to checkout a specific commit 

cc @fidencio

Reply:
yup, this will not work

```
Cloning into 'cloud-hypervisor'...
~/work/kata-containers/kata-containers/tools/packaging/kata-deploy/local-build/build/cloud-hypervisor/builddir/cloud-hypervisor ~/work/kata-containers/kata-containers/tools/packaging/kata-deploy/local-build/build/cloud-hypervisor/builddir
fatal: reference is not a tree: b0324f85571c441f840e9bdeb25410514a00bb74
make[1]: *** [tools/packaging/kata-deploy/local-build/Makefile:30: cloud-hypervisor-tarball-build] Error 128
```